docker/Dockerfile_local

```diff
@@ -10,7 +10,8 @@ WORKDIR '/faf'
 # Change owner of /faf, clean git and install dependences
 RUN chown -R faf:faf /faf && \
     git clean -dfx && \
-    dnf -y --setopt=strict=0 install $(./autogen.sh sysdeps) rpm-build && \
+    dnf -y install rpm-build && \
+    eval dnf -y --setopt=strict=0 install $(./autogen.sh sysdeps) && \
```

Comment:
Should this be ` eval dnf -y --setopt=strict=0 install '$(./autogen.sh sysdeps)'` because this way the $() is still evaluated before rpm-build is installed?

Reply:
rpm-build is installed before the ```eval dnf ... ```
Same as ``` dnf install package && dnf install package ```
The second dnf returns "Package package is already installed, skipping."

Without the quotes:
1. Run autogen.sh (For the ```$(autogen.sh sysdeps)``` substitution) 
2. Concatenate ```'dnf'  '-y'  '--setopt=strict=0'  'install'``` with the result of autogen.sh.
3. Run concatenated string as command
Runs ```dnf -y --setopt=strict=0 install ...``` as if it was typed.

```dnf -y --setopt=strict=0 install 'package >= 0.00'```

With quotes:
1. Concatenates ```'dnf'  '-y'  '--setopt=strict=0'  'install'  '$(./autogen.sh sysdeps)'``` into:
```dnf -y --setopt=strict=0 install $(./autogen.sh sysdeps)```
2. Run concatenated string as command
3. Run autogen.sh (For the ```$(autogen.sh sysdeps)``` substitution) 
4. Run ```dnf -y --setopt=strict=0 install ...``` with result of autogen.sh

``` dnf -y --setopt=strict=0 install 'package' '>=' '0.00' ```

The quotes would make the eval useless.